### PR TITLE
Enhance SortEvent Command

### LINKED ISSUE #109: As a user, I can sort my events
As a user, I can sort my events based on certain order (e.g. based on descriptions, timings) so that the event management is easier.

### PR DESCRIPTION
**Pull request by solkim-83**
1. Implement EventComparator class for better organisation and clarity 
2. Handle exception where sort -e (random index other than 1 or 2) is keyed in as an input

### CODE REVIEW

**Comment by solkim-83** [comment]
Also added an assertion statement for AddEventCommandParser in the same PR. Will be more mindful about the PR and will separate the implementations into different PR for clarify and organisation. Thanks.

File: src/main/java/seedu/address/logic/comparators/EventComparator.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    private static final Comparator<Event> TIME_COMPARATOR = new Comparator<Event>() {
+        @Override
+        public int compare(Event o1, Event o2) {
+            return o1.getTime().time.compareTo(o2.getTime().time);
+        }
+    };
+
+
+    /**
+     * Returns the appropriate comparator for the model manager to sort with
```
Comment (LinkedInk) [inline src/main/java/seedu/address/logic/comparators/EventComparator.java:27]: Java doc ends abruptly

Comment (solkim-83) [reply src/main/java/seedu/address/logic/comparators/EventComparator.java]: Edited. thanks.

**Review by LinkedInk — APPROVED** [review]
lgtm

**Pull request merged**